src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -322,17 +322,20 @@ private void initializeActiveRegionEvaluationGenotyperEngine() {
      * @return a VCF or GVCF writer as appropriate, ready to use
      */
     public VariantContextWriter makeVCFWriter( final String outputVCF, final SAMSequenceDictionary readsDictionary,
-                                               final boolean createOutputVariantIndex, final boolean  createOutputVariantMD5 ) {
+                                               final boolean createOutputVariantIndex, final boolean sitesOnlyMode,
+                                               final boolean  createOutputVariantMD5 ) {
```

Comment:
It would be slightly more logical if the ordering of the boolean args were: `createOutputVariantIndex, createOutputVariantMD5, sitesOnlyMode` instead